api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -44,8 +42,6 @@
 	
 	public static final long serialVersionUID = 4353L;
 
-	private static final Logger log = LoggerFactory.getLogger(PersonName.class);
```

Comment:
Removing this isn't required as a matter of fact that its used here 
https://github.com/openmrs/openmrs-core/blob/c4b7108dfd2aaf39e42fc02d73c07893bf3c08b0/api/src/main/java/org/openmrs/PersonName.java#L407

That's why this file isn't compiling. 

cc: @dkayiwa

Reply:
@samuelmale So what should I do now?